crates/cli/src/exec.rs

```diff
@@ -70,13 +63,17 @@ pub fn exec_setup(
 
     // Setup ZkWasm Vkey
     {
-        let vk_path = &output_dir.join(format!("{}.{}.vkey.data", prefix, 0));
+        let vk_path = &param_dir.join(format!("{}.vkey.data", prefix));
 
         if vk_path.exists() {
             info!("Found Verifying at {:?}", vk_path);
         } else {
             info!("Create Verifying to {:?}", vk_path);
-            let loader = ZkWasmLoader::<Bn256>::new(zkwasm_k, wasm_binary, phantom_functions)?;
+            let loader = ZkWasmLoader::<Bn256, ExecutionArg, StandardHostEnvBuilder>::new(
```

Comment:
'DefaultHostEnvBuilder' from `app_builder.rs` and 'StandardHostEnvBuilder' should be unified.

Reply:
Alternatively, you might use generic type.